parser.y

```diff
@@ -1478,13 +1481,37 @@ RecoverTableStmt:
  *      SPLIT TABLE table_name INDEX index_name BY (val1...),(val2...)...
  *
  *******************************************************************/
-SplitIndexRegionStmt:
-	"SPLIT" "TABLE" TableName "INDEX" IndexName "BY" ValuesList
+SplitRegionStmt:
+	"SPLIT" "TABLE" TableName SplitOption
 	{
-		$$ = &ast.SplitIndexRegionStmt{
+		$$ = &ast.SplitRegionStmt{
 			Table: $3.(*ast.TableName),
-			IndexName: $5.(string),
-			ValueLists: $7.([][]ast.ExprNode),
+			SplitOpt: $4.(*ast.SplitOption),
+		}
+	}
+|	"SPLIT" "TABLE" TableName "INDEX" Identifier SplitOption
+	{
+		$$ = &ast.SplitRegionStmt{
+			Table: $3.(*ast.TableName),
+			IndexName: model.NewCIStr($5),
+			SplitOpt: $6.(*ast.SplitOption),
+		}
+	}
+
+
+SplitOption:
+	"MIN" RowValue "MAX" RowValue "NUM" NUM
```

Comment:
Since `min` and `max` are not reserved keywords, IMHO it's not a good idea to use those 2 words here. @morgo PTAL

Reply:
@morgo PTAL

Reply:
what do you think about using `BETWEEN` here. i.e.:
`SPLIT TABLE [table_name] INDEX [index_name] BETWEEN min_value AND max_value REGIONS 5;`

This is more sql-like, since using min and max as function-like syntax feels weird to sql-writers, since this would usually be non-[sargable](https://dba.stackexchange.com/questions/162263/what-does-the-word-sargable-really-mean).

Reply:
done.